### PR TITLE
[15.0][MIG] sale_coupon_criteria_order_based: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT38306

@chienandalu @ernestotejeda please review :)

### CODE REVIEW

File: sale_coupon_criteria_order_based/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This module add and option to configure your coupon conditions based on
+the Sales Order values.
```
Comment (chienandalu) [inline sale_coupon_criteria_order_based/readme/DESCRIPTION.rst]:
```suggestion
This module adds an option to configure your coupon conditions based on
the Sales Order values.
```

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Review by ernestotejeda — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Thanks! :+1:

**Pull request merged**